src/utils/isReactComponentClass.js

```diff
@@ -56,11 +56,13 @@ export default function isReactComponentClass(
       classDeclaration = path.parentPath.value;
     }
     
-    if (classDeclaration.leadingComments && classDeclaration.leadingComments.length > 0) {
-      var matchedComments = classDeclaration.leadingComments.filter(function(comment) { return comment.value.match(/(@extends React.Component)/) });
-      if (matchedComments.length > 0) {
-        return true;
-      }
+    if (classDeclaration &&
+      classDeclaration.leadingComments &&
+      classDeclaration.leadingComments.length &&
```

Comment:
No need for the length check, since `some` will handle an empty array just fine.